examples/30_extended/study_tutorial.py

```diff
@@ -78,18 +80,40 @@
 
 openml.config.start_using_configuration_for_example()
 
-# Very simple classifier which ignores the feature type
+# Model that can handle missing values
+from sklearn.experimental import enable_hist_gradient_boosting
+from sklearn.ensemble import HistGradientBoostingClassifier
+
+
+# Helper functions to return required columns for ColumnTransformer
+def cont(X):
+    return X.dtypes != "category"
+
+def cat(X):
+    return X.dtypes == "category"
+
+
+cat_imp = make_pipeline(SimpleImputer(strategy="most_frequent"), OneHotEncoder(handle_unknown="ignore", sparse=False))
+ct = ColumnTransformer([
+    ("cat", cat_imp, cat),
+    ("cont", FunctionTransformer(lambda x: x, validate=False), cont)
+])
 clf = sklearn.pipeline.Pipeline(
     steps=[
-        ("imputer", sklearn.impute.SimpleImputer()),
-        ("estimator", sklearn.tree.DecisionTreeClassifier(max_depth=5)),
+        ("transform", ct),
+        ("estimator", HistGradientBoostingClassifier()),
     ]
 )
 
 suite = openml.study.get_suite(1)
 # We'll create a study with one run on three random datasets each
 tasks = np.random.choice(suite.tasks, size=3, replace=False)
 run_ids = []
+# 391 is all numeric
```

Comment:
Could you please drop those comments again?

Reply:
yes!